src/riscv.h

```diff
@@ -125,6 +125,9 @@ riscv_word_t rv_get_reg(struct riscv_t *, uint32_t reg);
 /* system call handler */
 void syscall_handler(struct riscv_t *rv);
 
+/* breakpoint exception handler */
+void breakpoint_handler(struct riscv_t *rv);
```

Comment:
In order to differentiate breakpoints in the GDB stub, rename to `ebreak_handler`.